<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClient</class>
 <widget class="QMainWindow" name="AddClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="1">
     <widget class="QLabel" name="labelStory">
      <property name="text">
       <string>Story</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="linePassCode"/>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineName"/>
    </item>
    <item row="2" column="3">
     <widget class="QDateEdit" name="dateEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="lineSavings"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelBirth">
      <property name="text">
       <string>Birth</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLineEdit" name="lineStory"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelPassCode">
      <property name="text">
       <string>Passport Code</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="linePhone"/>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="labelIncome">
      <property name="text">
       <string>Income </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelSurname">
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="lineIncome"/>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="labelSavings">
      <property name="text">
       <string>Savings</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineSurname">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="labelPhone">
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="4">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
